dub.json

```diff
@@ -43,11 +43,13 @@
         "submodules/agora/source/agora/utils/Test.d",
         "submodules/agora/source/agora/utils/TxBuilder.d",
         "submodules/agora/source/agora/utils/Utility.d",
-        "submodules/agora/source/agora/utils/WellKnownKeys.d"
+        "submodules/agora/source/agora/utils/WellKnownKeys.d",
+        "submodules/dyaml/source/dyaml/loader.d"
```

Comment:
This is not needed. We only use that hack in Agora because otherwise we pull in dependency to SCP.

Reply:
Ah is this why there are errors in the CI?

Reply:
Ye[